VMEncryption/main/HttpUtil.py

```diff
@@ -46,20 +46,26 @@ def __init__(self, logger):
     snapshot also called this. so we should not write the file/read the file in this method.
     """
 
-    def Call(self, method, http_uri, data, headers):
+    def Call(self, method, http_uri, data, headers, use_https=True):
```

Comment:
One thing to check in testing:  Does wire server have certain TLS version requirements?   If so there may be outdated older dual pass Linux virtual machines that could have succeeded with HTTP but will not succeed here.